pkgs/development/mobile/cocoapods/update

```diff
@@ -5,8 +5,16 @@ set -e
 
 cd "$(dirname "${BASH_SOURCE[0]}")"
 
+PLATFORM=$(bundler platform | head -n 1 | cut -d ":" -f 2 | cut -c 2-)
 rm -f Gemfile.lock Gemfile-beta.lock
+
 bundler lock
```

Comment:
Let's force a platform-independent lock:
```suggestion
BUNDLE_FORCE_RUBY_PLATFORM=1 bundler lock
```
with this variable, the other changes to this file aren't needed

Reply:
That's neat. Much better than using the flags.

Reply:
Thank you, I didn't know that environment variable existed!